Google.Api.Generator.Tests/ProtoTests/CommonResource/CommonResource.proto

```diff
@@ -0,0 +1,33 @@
+﻿syntax = "proto3";
+
+package testing.commonresource;
+
+option csharp_namespace = "Testing.CommonResource";
+
+import "google/api/client.proto";
+import "google/api/resource.proto";
+
+service CommonResource {
+  option (google.api.default_host) = "commonresource.example.com";
+
+  rpc MethodWithCommonResource(Request) returns(Response) {
+  	option (google.api.method_signature) = "project_name";
+  }
+}
+
+// This is defined as a common reosurce for C#, so will not have a class generated for it.
```

Comment:
Nit: reosurce => resource

(I know you only included this to check that I was paying attention.)

Reply:
:)
Done